Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -21,7 +21,7 @@
 #include "VideoCommon/VideoConfig.h"
 
 static const u64 TEXHASH_INVALID = 0;
-static const int TEXTURE_KILL_THRESHOLD = 60;
+static const int TEXTURE_KILL_THRESHOLD = 64;		// Sonic the Fighters(part of the Sonic Gems Collection) runs at 64fps internally
```

Comment:
Yes and no. I used my stopwatch and 99 seconds ingame is 126-127 seconds. So i guess the arcade machine runs at 64 fps.

But it seems there's some animation that takes 1 second on the arcade machine, which is  64 frames, and it's repeating.